pkg/git/git.go

```diff
@@ -91,12 +97,42 @@ func Open(path string, cfg Config) (*Git, error) {
 	return g, nil
 }
 
-func (g *Git) Increment(major, minor, patch, dev bool) (semver.Version, error) {
+func (g *Git) Increment(major, minor, patch, dev bool, rc bool) (semver.Version, error) {
 	newVersion, err := semver.Parse(g.highest.String())
 	if err != nil {
 		return semver.Version{}, fmt.Errorf("create version: %w", err)
 	}
 
+	if rc {
+		found := false
+		for i, pre := range newVersion.Pre {
```

Comment:
Would it be enough to just check the first one in the slice? As we only include the tag if the first one has the "rc" prefix